src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -183,9 +237,16 @@ class ProgressPane
                 return;
             }
 
-            // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
+            BufferCell[,] newRegion;
+            if (ExperimentalFeature.IsEnabled("PSAnsiProgress") && PSStyle.Instance.Progress.View == ProgressView.Minimal)
+            {
+                newRegion = new BufferCell[0, _content.Length];
+            }
+            else
+            {
+                newRegion = _rawui.NewBufferCellArray(_content, _ui.ProgressForegroundColor, _ui.ProgressBackgroundColor);
```

Comment:
This call works for other progress display preferences right? As in the rawUI object abstracts over the other progress display mechanisms?

If so, why is minimal progress handled differently at this layer? Shouldn't the rawUI class be responsible for this knowledge?

Reply:
The legacy progress rendering relies on a buffercell for rendering where each cell has a char, foreground color, and background color.  This model doesn't work for ANSI escape sequences.  However, there is also other logic in the code that relies on the buffercell.  I can add a comment where I create an empty buffercell, but use the second dimension to know later how many rows to clear.